src/components/buttons/Button.jsx

```diff
@@ -0,0 +1,159 @@
+// @flow
+
+import * as React from 'react';
+import type {Node} from 'react';
+import classNames from 'classnames';
+import {BUTTON_SIZE, BUTTON_TYPE} from './ButtonConsts';
+
+type ButtonTypeType =
+  | 'primary'
+  | 'primary-inverted'
+  | 'primary-blue'
+  | 'primary-mint'
+  | 'secondary'
+  | 'link-button'
+  | 'link-button-inverted'
+  | 'link-button-peach'
+  | 'link-button-mustard'
+  | 'destructive'
+  | 'warning'
+  | 'facebook';
+
+type ButtonSizeType =
+  | 'large'
+  | 'medium'
+  | 'small';
+
+export {BUTTON_SIZE, BUTTON_TYPE};
+
+export type ButtonPropsType = {
+  /**
+   * Children to be rendered inside Button
+   * @example <Button
+   *           icon={<Icon type={ICON_TYPE.ANSWER} color={ICON_COLOR.LIGHT} size={24} />}
```

Comment:
just `  <Button icon={<Icon type="answer" color="light" size={24} />} buttonType="facebook">` here and everywhere :)

Reply:
done, I hope I didn't miss anything.

Reply:
only one last place in Readme, `size={BUTTON_SIZE.SMALL}`

Reply:
@PinkaminaDianePie soooorrry, I am blind. should be fixed now. thx 🙂